test-workspace/tsc/vue3/#3765/main.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+	<Foo stroke-width="test" />
+	<component :is="Foo" stroke-width="test"></component>
+	<component :is="Foo" strokeWidth="test"></component>
+</template>
+
+<script setup lang="ts">
+import { defineComponent } from 'vue';
+
+const Foo = defineComponent({
+	props: {
+		strokeWidth: { type: String, required: true },
+	},
+});
+</script>
```

Comment:
I would think that it makes sense to update `test-workspace/tsc/vue3_strictTemplate/dataAttributes/main.vue` so that it includes all cases in one place.